include/dts-environment.sh

```diff
@@ -97,7 +108,7 @@ FLASH_CHIP_LIST="W25Q64BV/W25Q64CV/W25Q64FV W25Q64JV-.Q W25Q128.V..M"
 # Dasharo Supporters Entrance variables
 SE_credential_file="/etc/cloud-pass"
```

Comment:
If you want to rename all variables, then `SE` is also the one to be replaced with `DPP`

Reply:
Fixed here db23885453000db1f5f6f7a26249c768c87353cb.